tests/drivers/motor_driver/main.c

```diff
@@ -21,20 +21,36 @@
 #include <string.h>
 
 /* RIOT includes */
+#include "init_dev.h"
 #include "log.h"
 #include "motor_driver.h"
+#include "motor_driver_params.h"
 #include "test_utils/expect.h"
 #include "xtimer.h"
```

Comment:
```suggestion
#include "ztimer.h"
```

Reply:
if you like

Reply:
done

Reply:
did you push your changes?

Reply:
just pushed the fixes, was off for few days, sorry. ;)